src/pybind/mgr/telemetry/module.py

```diff
@@ -755,19 +758,37 @@ def handle_command(self, inbuf, command):
             return self.send(self.last_report, command.get('endpoint'))
 
         elif command['prefix'] == 'telemetry show':
-            report = self.compile_report(
-                channels=command.get('channels', None)
-            )
+            report = self.get_report(channels=command.get('channels', None))
             report = json.dumps(report, indent=4, sort_keys=True)
             if self.channel_device:
-               report += '\n \nDevice report is generated separately. To see it run \'ceph telemetry show-device\'.'
+                report += '\n \nDevice report is generated separately. To see it run \'ceph telemetry show-device\'.'
```

Comment:
Extra space.

Reply:
I fixed the indentation. We use a indentation of 4 spaces but this line was indented by 3. Or should I remove it here and commit it separately?

Reply:
Oh you're right. Please ignore my comment here.